website/release_announcement_drafts/v1.3.0.md

```diff
@@ -0,0 +1,28 @@
+We're excited to announce the v1.3.0 release of JBrowse Web! Some highlights of
+this release include:
+
+## In-app plugin store
+
+Building on the plugin store on our website in the last release, we're now
+excited to announce that plugins can be installed from within JBrowse Web!
+Plugins from our plugin store can now be installed with the click of a button.
+
+![Screenshot of plugin store](https://user-images.githubusercontent.com/19295181/117894133-8dcc7480-b270-11eb-89fc-779be2e517a3.png)
+![Screenshot of plugin store with MsaView plugin installed and open](https://user-images.githubusercontent.com/19295181/117894797-f9631180-b271-11eb-9c81-69c5aa6ae497.png)
+
+## Open local files
+
+JBrowse Web now has the ability for tracks to use files on your local hard
+drive. This is a great option if you want to visualize files you have locally
+without uploading them to a server. These files will need to be re-opened each
+time the app is opened or refreshed, but more robust handling of local files
+will be available when we release JBrowse Desktop.
+
+## Color by MM and MP/ML tags in BAM/CRAM
+
+The MM and MP/ML tags can be used to color alignments tracks by either base
+modifications or by methylation.
+
+![Screenshot of alignments track colored by methylation](https://user-images.githubusercontent.com/25592344/119381778-83ea3e80-bc7e-11eb-861d-ca39645a9eb1.png)
```

Comment:
could possibly combine both into a single screenshot ? e.g. https://raw.githubusercontent.com/GMOD/jbrowse-components/6ad23cd4381bfa3ed4617d60c3be0294cedfd654/website/static/img/alignment_modifications.png from #2007

Reply:
the menu items in screenshot are now outdated...went ahead and updated to use https://raw.githubusercontent.com/GMOD/jbrowse-components/7f0c725a929bb15a6adfcf3a155bf9dc5f717af7/website/static/img/alignments/modifications1.png

Reply:
Thanks!